docs/changelog.rst

```diff
@@ -2,6 +2,7 @@
 Changelog
 =========
 
+* :bug:`9586` Fix an issue where adding a Bitpanda exchange account would fail due to incorrectly requiring an API secret when Bitpanda only uses API keys.
```

Comment:
How many times do I have to repeat this? Changelog should be user facing. Not what you did.

```suggestion
* :bug:`9586` Adding a new Bitpanda exchange account should now work correctly again.
```